src/components/sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { useCategory } from "../../context";
+import "./sidebar.css";
+
+export default function Sidebar() {
+  const { categoryState, setCategoryState } = useCategory();
+
+  return (
+    <aside className="sidebar dui-util-spc-pad-1_6rem-s">
+      <div className="sidebar__sort-by">
+        <p className="dui-util-txt-sm dui-util-fw-bld dui-util-disp-inln-blk">
+          Sort By Quiz Count
+        </p>
+
+        <div>
+          {/* <!-- Radio Button Component Starts --> */}
+          <label className="dui-inp-radio-btn dui-util-txt-sm">
+            Low to High
+            <input
+              type="radio"
+              name="sort-by"
+              checked={categoryState.sortByQuizCount === "LOW_TO_HIGH"}
+              onChange={() =>
+                setCategoryState({
+                  type: "SORT_BY_QUIZ_COUNT",
+                  data: { sortByQuizCount: "LOW_TO_HIGH" },
+                })
+              }
+            />
+            <span className="dui-inp-radio-btn__checkmark"></span>
+          </label>
+
+          <label className="dui-inp-radio-btn dui-util-txt-sm">
+            High to Low
+            <input
+              type="radio"
+              name="sort-by"
+              checked={categoryState.sortByQuizCount === "HIGH_TO_LOW"}
+              onChange={() =>
+                setCategoryState({
+                  type: "SORT_BY_QUIZ_COUNT",
+                  data: { sortByQuizCount: "HIGH_TO_LOW" },
+                })
+              }
+            />
+            <span className="dui-inp-radio-btn__checkmark"></span>
+          </label>
+          {/* <!-- Radio Button Component Ends --> */}
+        </div>
```

Comment:
Here maybe you can make an array for all kind of filter and put those values in array and map them on this label. This way you can avoid code repetition.

Reply:
Done.